README.md

```diff
@@ -6,5 +6,13 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails is a server-side MVC framework that helps users quickly and easily build up the default structures for a web application.
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
Your diagram is missing a few key features - the Server and the Database, specifically.